src/Console/Addon.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * @copyright Copyright (C) 2021, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Console;
+
+use Console_Table;
+use Friendica\App;
+use Friendica\Content\Pager;
+use Friendica\Core\L10n;
+use Friendica\Core\Addon as AddonCore;
+use Friendica\Database\Database;
+use Friendica\Util\Strings;
+use RuntimeException;
+
+/**
+ * tool to manage addons on the current node
+ */
+class Addon extends \Asika\SimpleConsole\Console
+{
+	protected $helpOptions = ['h', 'help', '?'];
+
+	/**
+	 * @var App\Mode
+	 */
+	private $appMode;
+	/**
+	 * @var L10n
+	 */
+	private $l10n;
+	/**
+	 * @var Database
+	 */
+	private $dba;
+
+	protected function getHelp()
+	{
+		$help = <<<HELP
+console user - Modify addon settings per console commands.
+Usage
+	bin/console addon list all [-s|--start=0] [-c|--count=50] [-h|--help|-?] [-v]
+	bin/console addon list enabled [-s|--start=0] [-c|--count=50] [-h|--help|-?] [-v]
+	bin/console addon list disabled [-s|--start=0] [-c|--count=50] [-h|--help|-?] [-v]
```

Comment:
Please remove the unused parameters start and count from these three command synopses.